### PR TITLE
Feature/support for truncate table query parsing (#1)

### LINKED ISSUE #520: parse does not support `truncate table <tablename>` queries
```
import sql_metadata
sql_metadata.parser.Parser('truncate table foo.bar.baz').tokens
```
 results in 
```
----> 1 sql_metadata.parser.Parser('truncate table foo.bar.baz').tokens

File /turbo/tsantos/pyenvs/ni.bau_38/lib/python3.8/site-packages/sql_metadata/parser.py:179, in Parser.tokens(self)
    174 self._tokens = tokens
    175 # since tokens are used in all methods required parsing (so w/o generalization)
    176 # we set the query type here (and not in init) to allow for generalization
    177 # but disallow any other usage for not supported queries to avoid unexpected
    178 # results which are not really an error
--> 179 _ = self.query_type
    180 return tokens

File /turbo/tsantos/pyenvs/ni.bau_38/lib/python3.8/site-packages/sql_metadata/parser.py:122, in Parser.query_type(self)
    120 if self._query_type == "UNSUPPORTED":
    121     self._logger.error("Not supported query type: %s", self._raw_query)
--> 122     raise ValueError("Not supported query type!")
    123 return self._query_type

ValueError: Not supported query type!
```

### PR DESCRIPTION
**Pull request by T-Santos**
Resolves #520

### CODE REVIEW

File: test/test_truncate_table.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from sql_metadata import Parser
+from sql_metadata.keywords_lists import QueryType
+
+
+def test_truncate_table():
+    parser = Parser("TRUNCATE TABLE foo")
+    assert parser.query_type == QueryType.TRUNCATE
+    assert parser.tables == ["foo"]
+    assert parser.columns == []
```
Comment (macbre) [inline test/test_truncate_table.py:9]:
```suggestion
    assert parser.columns == []

```

Adding a new line to make `black` happy.

Comment (macbre) [reply test/test_truncate_table.py:9]: @T-Santos, can you commit the above suggestion?

**Review by macbre — APPROVED** [review]
LGTM. Thanks for the PR! One comment left regarding the code formatting.

**Comment by T-Santos** [comment]
@macbre please let me know if there's anything else needed from me to move this forward. thank you

**Pull request merged**